### ISSUE TITLE
Topic request: libsecp256k1-zkp

### ISSUE THREAD

**Issue by ysangkok** [bitcoinops/bitcoinops.github.io#557]
It would be nice if the glossary covered libsecp256k1-zkp and its history and features. [PR 117 with ECDSA adaptor signatures](https://github.com/ElementsProject/secp256k1-zkp/pull/117) was recently merged. But it would be nice if other special features were also covered.

**Comment by ysangkok** [comment]
If the topic is added, it would be nice if the "Adaptor signatures" topic would mention it. Right now, it says "This is also possible in ECDSA but it requires novel algorithms that are either comparatively slow or require additional security assumptions". Is that still true?

If the topic were added, it would be nice if the [PTLC topic](https://bitcoinops.org/en/topics/ptlc/) would mention that it is possible to build PTLCs for ECDSA using libsecp256k1-zkp.

**Comment by harding** [comment]
@ysangkok Thanks for the suggestion!  I'm hesitant to create topic pages for specific software.  We did create a page for HWI in our initial batch of topics and I've been kind of regretting it since (and I think I'd like to remove the HWI page at some point).  It feels weird to me mixing topics about pure ideas with topics for specific implementations.  It also gets confusing when we mention the software (e.g. a merged PR for HWI in a newsletter) but don't add a link to its topic page.

Re the signature adaptors page, I think it already describes what PR 112 does in the last sentence of the main text:

> This is also possible in ECDSA but it requires novel algorithms that are either comparatively slow or require additional security assumptions. Instead, **an alternative scheme for adaptor signatures exists for Bitcoin that uses 2-of-2 OP_CHECKSIG multisig; this is less efficient and possibly less private—but arguably simpler and safer than multiparty ECDSA.**

Re the PTLC page, I think that also currently describes the state of the art with regard to ECDSA in the last paragraph, basically that it's possible but it seems most developers are waiting on the availability of schnorr to put much time into it.

I'll see if I can work a mention of the PR 119 merge into next week's newsletter; that way I can add it to the various topic pages and so it'll be there for anyone who looks at the list of mentions.

Thanks again!

**Comment by ysangkok** [comment]
Thanks, I see that latest issue covered it.
